consensus/polybft/fsm.go

```diff
@@ -195,7 +196,14 @@ func (f *fsm) stateTransactions() []*types.Transaction {
 			}
 
 			txns = append(txns,
-				createStateTransactionWithData(f.config.SidechainBridgeAddr, inputData))
+				createStateTransactionWithData(f.config.StateReceiverAddr, inputData))
+
+			// since proposer does not execute Validate (when we see the commitment to register in state transactions)
```

Comment:
This assignment should be removed. Instance field is changed outside this function, this function should be used just to retrieve all the state transactions

Reply:
Fixed [dad6eba](https://github.com/0xPolygon/polygon-edge/pull/796/commits/dad6ebaa45aa22820d59d8f3f3efde388220bbc9)